app/views/layouts/_header.html.erb

```diff
@@ -121,48 +121,9 @@
                 <li><a href="/logoutRemotely">Logout from all devices</a></li>
               </ul>
             <% else %>
-              <a class="hidden-sm" href="#" class="dropdown-toggle" data-toggle="dropdown">
-                 <%= t('layout._header.login.login_title') %>
+              <a href="/login?return_to=<%= params[:return_to] || request.path %>">
+                <%= t('layout._header.login.login_title') %>
               </a>
-              <ul id="login-dropdown" class="dropdown-menu" style="width:245px; color: white;">
```

Comment:
You have deleted this by mistake I guess.

Reply:
Actually, it's w.r.t. this comment : https://github.com/publiclab/plots2/pull/3879#issuecomment-435575944

Though in case you feel we need it, I can try adding it back.

Reply:
I see. 

When Jeff and I implemented this OAuth system we made this working on the localhost, unstable, stable and on the production.
So, once you deploy this onto the unstable and check the functionality for all the providers, ping me. I will check it.

Thanks for the awesome hard work.

Reply:
This might help you
https://github.com/publiclab/plots2/blob/master/doc/OMNIAUTH.md

Reply:
No need to check for all providers on the development. Just create anyone say github and check it on local.
https://github.com/publiclab/plots2/issues/2856 
On unstable we need to check it on all providers.
My insights says that your solution will work. I did the same solution in February.

Reply:
> https://github.com/publiclab/plots2/blob/master/doc/OMNIAUTH.md

Thanks, I understand the process now!

>  On unstable we need to check it on all providers.

Oh cool

>  My insights says that your solution will work. I did the same solution in February.

Oh, so did it get dropped mistakenly in refactoring or something ? 


cool, I'll try it out.